performance_curves/performance_curve_bundle.py

```diff
@@ -0,0 +1,48 @@
+from typing import Mapping
+from performance_curves.performance_curve import *
+
+
+class PerformanceCurveBundle:
+    """A bundle includes multiple performance curves that represent multiple methods (e.g. ranking systems, or ML models)
+    applied to the same test data and evaluated under the same metric"""
+    def __init__(self,
+                 y_true: np.ndarray,
+                 method_scores: Mapping[str, np.ndarray],
+                 metric: Metric,
+                 num_bins: Optional[int] = None,
+                 order_descending: bool = True,
+                 make_perfect: bool = True,
+                 make_random: bool = True,
+                 num_trials: Optional[int] = 1):
```

Comment:
It doesn't seem like `num_trials` can be `None`. Is that correct? If so, then it is not `Optional`.

Reply:
Good catch! `num_trials` is changed to a non-optional argument.